x-pack/legacy/plugins/code/server/lsp/proxy.ts

```diff
@@ -93,7 +93,7 @@ export class LanguageServerProxy implements ILanguageServerHandler {
       return Promise.resolve(this.clientConnection);
     } else {
       return new Promise<MessageConnection>((resolve, reject) => {
-        this.eventEmitter.on('error', error =>
```

Comment:
The root cause is here.  I didn't know `eventEmitter.emit('error', error)` will throw a exception.  Node js doc never mentioned this.